src/PaymentPart/Output/FpdfOutput/FpdfOutput.php

```diff
@@ -0,0 +1,212 @@
+<?php
+
+
+namespace Sprain\SwissQrBill\PaymentPart\Output\FpdfOutput;
+
+use Sprain\SwissQrBill\PaymentPart\Output\AbstractOutput;
+use Sprain\SwissQrBill\PaymentPart\Output\Element\OutputElementInterface;
+use Sprain\SwissQrBill\PaymentPart\Output\Element\Placeholder;
+use Sprain\SwissQrBill\PaymentPart\Output\Element\Text;
+use Sprain\SwissQrBill\PaymentPart\Output\Element\Title;
+use Sprain\SwissQrBill\PaymentPart\Output\FpdfOutput\Template\QrBillFooter;
+use Sprain\SwissQrBill\PaymentPart\Output\OutputInterface;
+use Sprain\SwissQrBill\PaymentPart\Translation\Translation;
+use Sprain\SwissQrBill\QrBill;
+use Sprain\SwissQrBill\QrCode\QrCode;
+
+final class FpdfOutput extends AbstractOutput implements OutputInterface
+{
+    /** @var QrBillFooter */
+    private $fpdf;
+
+    /** @var float */
+    private $amountLS = 0;
+
+    public function __construct(
+        QrBill $qrBill,
+        string $language,
+        QrBillFooter $fpdf
+    ) {
+        parent::__construct($qrBill, $language);
+        $this->fpdf = $fpdf;
+        $this->setQrCodeImageFormat(QrCode::FILE_FORMAT_PNG);
+    }
+
+    public function getPaymentPart()
+    {
+        $this->fpdf->SetAutoPageBreak(false);
+        $this->addSeparatorContentIfNotPrintable();
+        $this->addReceiptPart();
+        $this->addPaymentPart();
+    }
+
+    private function addSeparatorContentIfNotPrintable()
+    {
+        if (!$this->isPrintable()) {
+            $this->fpdf->SetLineWidth(0.1);
+            $this->fpdf->SetDash(1.25, 1.25);
+            $this->fpdf->Line(2, 193, 208, 193);
+            $this->fpdf->Line(62, 193, 62, 296);
+            $this->fpdf->SetFont('helvetica', '', 7);
+            $this->fpdf->SetY(189.6);
+            $this->fpdf->MultiCell(0, 0, utf8_decode(Translation::get('separate', $this->language)), '', 'C');
+        }
+    }
+
+    private function addReceiptPart()
+    {
+        // Title
+        $this->fpdf->SetFont('helvetica', 'B', 11);
```

Comment:
I like how `TcPdfOutput` puts all units, like font sizes etc, into class constants. We could do the same here, even using the same namings. This will make it easier to adjust should the qr bill standard change in the future.

Reply:
Ok I will do that.